src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -215,86 +166,64 @@ async function importDotnetJs(startOptions: Partial<WebAssemblyStartOptions>): P
     }
   }
 
-  // For consistency with WebAssemblyResourceLoader, we only enforce SRI if caching is allowed
```

Comment:
Does runtime behave in the same way W.R.T. SRI/caching ?

Reply:
I don't have the hash for `dotnet.js` anymore